components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -74,3 +74,11 @@
 .errorContainer {
   padding: 16px;
 }
+
+.alertCenter {
+  --leo-alert-center-width: calc(100% - 32px); /* remove scrollbar space */
```

Comment:
permanent scrollbar, or overlaid sometimes-visible scrollbar? Seems dangerous to hardcode 32px if we want it to appear full-width. But I think I just need more info.

Reply:
sometimes-visible scrollbar

Reply:
so does it look off-center when the scrollbar isn't visible or is always visible (windows, linux, macos with that setting)?